engine/remote-components/src/main/java/org/datacleaner/components/remote/RemoteTransformer.java

```diff
@@ -120,9 +125,13 @@ public OutputColumns getOutputColumns() {
                         try {
                             outCols.setColumnType(i, Class.forName(colSpec.type));
                         } catch (ClassNotFoundException e) {
-                            // TODO: what to do with data types - classes that are not on our classpath?
-                            // Provide it as pure JsonNode?
-                            outCols.setColumnType(i, JsonNode.class);
+                            Class type;
+                            if (isOutputColumnEnumeration(colSpec.schema)) {
+                                type = EnumerationValue.class;
```

Comment:
You are right. The String will be better. I will do changes, and I will do new pull request.